app/controllers/api/v1/api_controller.rb

```diff
@@ -8,6 +8,6 @@ def handle_error(e)
   end
 
   def current_organization
-    @current_organization ||= Organization.find_by!(token: request.headers['token'])
+    @current_organization ||= Organization.find_by!(token: request.headers['Token'])
```

Comment:
make an API helper, e.g.: `request_token` or something like that so that if header changed it wouldn't make complications in synsing all places